src/util/cliUtil.js

```diff
@@ -0,0 +1,28 @@
+const findUp = require('find-up');
+const fs = require('fs-extra-promise');
+const path = require('path');
+
+const SITE_CONFIG_NAME = 'site.json';
+
+module.exports = {
+  findRootFolder: (userSpecifiedRoot, siteConfigPath = SITE_CONFIG_NAME) => {
+    if (userSpecifiedRoot) {
+      const resolvedUserSpecifiedRoot = path.resolve(userSpecifiedRoot);
+      const expectedConfigPath = path.join(resolvedUserSpecifiedRoot, siteConfigPath);
+      if (!fs.existsSync(expectedConfigPath)) {
+        throw new Error(`Config file not found at user specified root ${resolvedUserSpecifiedRoot}`);
+      }
+      return resolvedUserSpecifiedRoot;
+    }
+
+    const currentWorkingDir = process.cwd();
+    // Enforces findUp uses value of process.cwd() to determine starting dir
+    // This allows us to define starting dir when testing by mocking process.cwd()
+    const foundConfigPath = findUp.sync(siteConfigPath, { cwd: currentWorkingDir });
+    if (!foundConfigPath) {
+      throw new Error(`No config file found in parent directories of ${currentWorkingDir}`);
```

Comment:
![error](https://user-images.githubusercontent.com/31084833/51424562-7f26a300-1c0a-11e9-898c-668781f42da2.PNG)

- Serving on my local desktop does not throw the error written here.